packages/http-client-csharp/package.json

```diff
@@ -69,12 +69,12 @@
     "@types/node": "~22.12.0",
     "@typespec/compiler": "1.3.0",
     "@typespec/http": "1.3.0",
-    "@typespec/http-specs": "0.1.0-alpha.25",
+    "@typespec/http-specs": "0.1.0",
```

Comment:
Do not update here lib version here as http-specs": "0.1.0" is not published yet.